tests/test-definition.yaml

```
@@ -0,0 +1,9 @@
+---
+dimensions:
+  - name: edc
+    values:
+      - a  # value not used
```

Why not used ? You could put the version and use it in the `03-install-edc.yaml`
```suggestion
      - 77c6e17-stackable0.0.0-dev
```

---

good point, I didn't thin of that. Fixed.